integration/cli/build.nonconflict.test.ts

```diff
@@ -0,0 +1,187 @@
+import * as path from "node:path";
+import * as fs from "node:fs/promises";
+import { kind } from "kubernetes-fluent-client";
+import { Workdir } from "../helpers/workdir";
+import * as time from "../helpers/time";
+import * as pepr from "../helpers/pepr";
+import * as resource from "../helpers/resource";
+import * as file from "../helpers/file";
+
+const FILE = path.basename(__filename);
+const HERE = __dirname;
+
+describe("build", () => {
+  const workdir = new Workdir(`${FILE}`, `${HERE}/../testroot/cli`);
+
+  beforeAll(async () => {
+    await workdir.recreate();
+  });
+
+  describe("builds a module", () => {
+    const id = FILE.split(".").at(1);
+    const mod = `${workdir.path()}/${id}`;
+
+    beforeAll(async () => {
+      await fs.rm(mod, { recursive: true, force: true });
+      const argz = [
+        `--name ${id}`,
+        `--description ${id}`,
+        `--errorBehavior reject`,
+        "--confirm",
+        "--skip-post-init",
+      ].join(" ");
+      await pepr.cli(workdir.path(), { cmd: `pepr init ${argz}` });
+      await pepr.tgzifyModule(mod);
+      await pepr.cli(mod, { cmd: `npm install` });
+    }, time.toMs("2m"));
+
+    describe("using non-conflicting build override options", () => {
+      const entryPoint = "pepr2.ts";
+      const customImage = "pepr:override";
+      const outputDir = `${mod}/out`;
+      const timeout = 11;
+      const withPullSecret = "shhh";
+      const zarf = "chart";
+
+      let packageJson;
+      let uuid: string;
+
+      it(
+        "builds",
+        async () => {
+          await fs.rename(`${mod}/pepr.ts`, `${mod}/${entryPoint}`);
+
+          const argz = [
+            `--entry-point ${entryPoint}`,
+            `--custom-image ${customImage}`,
+            `--output-dir ${outputDir}`,
+            `--timeout ${timeout}`,
+            `--withPullSecret ${withPullSecret}`,
+            `--zarf ${zarf}`,
+          ].join(" ");
+          const build = await pepr.cli(mod, { cmd: `pepr build ${argz}` });
+
+          expect(build.exitcode).toBe(0);
+          expect(build.stderr.join("").trim()).toBe("");
+          expect(build.stdout.join("").trim()).toContain("K8s resource for the module saved");
+
+          packageJson = await resource.oneFromFile(`${mod}/package.json`);
+          uuid = packageJson.pepr.uuid;
+        },
+        time.toMs("1m"),
+      );
+
+      const getDepConImg = (deploy: kind.Deployment, container: string): string => {
+        return deploy!.spec!.template!.spec!.containers.filter(f => f.name === container).at(0)!
+          .image!;
+      };
+
+      it("--entry-point, works", async () => {
+        // build would fail if given entrypoint didn't exist, so... no-op test!
+      });
+
+      it("--output-dir, works", async () => {
+        const dist = `${mod}/dist`;
+        expect(await file.exists(dist)).toBe(false);
+
+        const out = outputDir;
+        expect(await file.exists(out)).toBe(true);
```

Comment:
```suggestion
        expect(await file.exists(outputDir)).toBe(true);
```

Seems like we don't need to assign the higher-scoped variable to a local const since we use it for an assertion.